docs/includes/usage-examples/FindManyTest.php

```diff
@@ -33,11 +34,18 @@ public function testFindMany(): void
             ],
         ]);
 
-        // begin-find
+        // begin-eloquent-find
         $movies = Movie::where('runtime', '>', 900)
             ->orderBy('_id')
             ->get();
-        // end-find
+        // end-eloquent-find
```

Comment:
You must keep the previous assertion, otherwise your query is not tested.

```suggestion
        // end-eloquent-find
        
        $this->assertEquals(2, $movies->count());
```

Reply:
I'll go through an re-add the tests where I missed them!